### PR TITLE
TEMP: Skip campaigns created on staging w/auction attributes.

### PR DESCRIPTION
**Pull request by dwick**
:eyeglasses: @zeantsoi

### CODE REVIEW

File: reddit_adzerk/adzerkpromote.py [hunk 1]
```diff
@@ -488,6 +488,11 @@ def _update_adzerk(link, campaign):
         az_campaign = update_campaign(link, az_advertiser)
         az_creative = update_creative(link, az_advertiser)
 
+        # Temporarily skip new (auction/fix cpm) campaigns during
+        # testing.
+        if campaign and not hasattr(campaign, "bid", False):
```
Comment (zeantsoi) [inline reddit_adzerk/adzerkpromote.py]: Only accepts two arguments.

**Comment by zeantsoi** [comment]
:nail_care:

**Comment by dwick** [comment]
:haircut:

**Comment by zeantsoi** [comment]
:fish:

**Pull request merged**